src/schema/v1/fields/money.ts

```diff
@@ -105,6 +109,37 @@ const money = ({ name, resolve }) => ({
   }),
 })
 
+export const Price = new GraphQLObjectType<any, ResolverContext>({
+  name: "Price",
+  fields: {
+    minor: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: "An amount of money expressed in minor units (like cents).",
+      resolve: ({ cents }) => cents,
+    },
+    currencyCode: {
+      type: new GraphQLNonNull(GraphQLString),
+      description:
+        "The ISO-4217 alphabetic currency code, as per https://en.wikipedia.org/wiki/ISO_4217",
+      resolve: ({ currency }) => currency,
+    },
+    display: {
+      type: GraphQLString,
+      description: "A pre-formatted price.",
+    },
+    major: {
+      type: new GraphQLNonNull(GraphQLFloat),
+      description:
+        "An amount of money expressed in major units (like dollars).",
+      resolve: ({ cents, currency }) => {
+        const factor = currencyCodes[currency.toLowerCase()].subunit_to_unit
+        // TODO: Should we round or used a fixed precision?
```

Comment:
Is this worth ticketing after this PR merges? I'm not sure of the answer to this question but if it's important, might be worth getting into Jira.

Reply:
i think in reality we probably wouldn't deal with fractional parts of a major unit here too often for artwork prices, but using this type more generally (to represent transaction amounts which might include tax or other fees), it would be possible to come up with `100.25`, or whatever.

So most likely we shouldn't round (or need to offer rounding) here, and clients can do that if needed.